### PR TITLE
July 29 updates

### PR DESCRIPTION
**Pull request by alexvandesande**

### CODE REVIEW

**Comment by frozeman** [comment]
Do you want to show the icon under the read banner when its not agreed yet in the cli section?
I would hide them.

File: views/content/ether.md [hunk 1]
```diff
@@ -72,7 +91,7 @@ Ethereum would never be possible without bitcoin—both the technology and the c
 
 There are many ways in which you can use Bitcoins within the Ethereum ecosystem:
 
-* **Trade BTC for ETH:** we are working with multiple exchanges to make the exchanging of ether and bitcoins as easy and seamless as possible. If so desired one could trade a bitcoins for ether with the purpose of executing contracts and trade it back immediately in order to keep their value pegged and secured by the bitcoin network. As those exchanges go live, we will list them here.
+* **Trade BTC for ETH:** multiple third party companies are working to make the exchanging of ether and bitcoins as easy and seamless as possible. If so desired one could trade a bitcoins for ether with the purpose of executing contracts and trade it back immediately in order to keep their value pegged and secured by the bitcoin network.
```
Comment (frozeman) [inline views/content/ether.md]: You could mention Kraken and Gatecoin, they have us already listed

**Comment by frozeman** [comment]
Some notes:

- I'm not sure i f `~/mypath/` for matador and genesis block works in the CLI

File: views/content/ether.md (hunk 1 above)
Comment (obscuren) [inline views/content/ether.md]: We do not list exchanges

**Pull request merged**